hw2/05-form.js

```diff
@@ -1 +1,24 @@
 // Add your code here
+function printToConsole(event) {
+  event.preventDefault();
+
+  const name = document.getElementById('fullName').value;
+  const email = document.getElementById('email').value;
+  const registration = document.getElementById('registration').value;
+  const course1 = document.getElementById('course1').checked;
+  const course2 = document.getElementById('course2').checked;
+  const course3 = document.getElementById('course3').checked;
+  const feedback = document.getElementById('feedback').value;
```

Comment:
Instead of querying the DOM multiple times for the form elements and their value, why not just use [form.elements](https://eloquentjavascript.net/2nd_edition/18_forms.html) to query the DOM once.